Linking.md

```diff
@@ -262,30 +288,35 @@ offset of the immediate within the code section.
 The same technique applies for all function calls whether the function is
 imported or defined locally.
 
+The Wasm object files shall have imports corresponding to all undefined
+symbols, and all defined symbols have an associated export, including
+those with hidden visibility or internal (local) linkage.
```

Comment:
This sounds like it applies to all symbols, but you mean all function symbols I think?

Reply:
Ah yes, I'll clean that up. It's implicit in the fact that it's in the Functions section!